pandas/core/internals/managers.py

```diff
@@ -1890,7 +1891,28 @@ def _consolidate(blocks):
     return new_blocks
 
 
-def _maybe_compare(a, b, op):
+def _maybe_compare(a, b, regex=False):
```

Comment:
can you make this slightly more verbose name, can you move to pandas/core/ops.py (cc @jbrockmendel good location)?

Reply:
Do we anticipate using it elsewhere?  If not I'd leave it here at least for now.

Reply:
like name it ``_compare_or_regex_match`` ?